projects/storage/src/services/storage.service.ts

```diff
@@ -0,0 +1,56 @@
+import {Inject, Injectable} from '@angular/core';
+import {LOCAL_STORAGE, WINDOW} from '@ng-web-apis/common';
+
+// @dynamic
+@Injectable({
+    providedIn: 'root',
+})
+export class StorageService implements Storage {
```

Comment:
May be LocalStorageService?

Reply:
I'd like to keep it short. This doesn't make sense for `sessionStorage` since this whole behavior is only possible with `localStorage`